SamplesDashboard/SamplesDashboard/ClientApp/src/components/DetailList.tsx

```diff
@@ -0,0 +1,186 @@
+﻿import * as React from 'react';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { DetailsList, DetailsListLayoutMode, Selection, IColumn, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import { MarqueeSelection } from 'office-ui-fabric-react/lib/MarqueeSelection';
+import { ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react/lib/ScrollablePane';
+import { Sticky, StickyPositionType } from 'office-ui-fabric-react/lib/Sticky';
+import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
+import { mergeStyles, mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { Announced } from 'office-ui-fabric-react/lib/Announced';
+
+const detailListClass = mergeStyles({
+    display: 'block',
+    marginBottom: '10px'
+
+});
+const classNames = mergeStyleSets({
+    wrapper: {
+        height: '80vh',
+        position: 'relative',
+        display: 'flex',
+        flexWrap: 'wrap'
+    }
+});
+
+export interface IListDataItem {
+    key: number;
+    name: string;
+    owner: string;
+    status: string;
+    language: string;
+    pullRequests: number;
+    issues: number;
+    stars: number;
+    featureArea: string;
+    securityAlerts: string;
+}
+
+export interface IListDataState {
+    columns: IColumn[];
+    items: IListDataItem[];
+    selectionDetails: string;
+    announcedMessage?: string;
+}
+
+export default class ListData extends React.Component<{}, IListDataState> {
+    private _selection: Selection;
+    private _allItems: IListDataItem[];
+
+    constructor(props: {}) {
+        super(props);
+
```

Comment:
Check out https://github.com/OfficeDev/office-ui-fabric-react/wiki/Using-icons.

You the missing icons will show once you this line here
```js 
initializeIcons(/* optional base url */);
```